x/gravity/keeper/grpc_query_get_all_bridge_transfer_limits.go

```diff
@@ -0,0 +1,41 @@
+package keeper
+
+import (
+	"context"
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	keeperutil "github.com/palomachain/paloma/util/keeper"
+	"github.com/palomachain/paloma/x/gravity/types"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+func (k Keeper) GetAllBridgeTransferLimits(
```

Comment:
FYI, I've been thinking a bit about API design on the keepers lately. The most convincing argument I've come across is to 

- start functions related to gRPC queries with `Query...`
- start functions on message server implementation with the verb, i.e. `Relay...`, `Start...`, etc...
- start simple getter functions on the keeper implementation with `Get`
- name more advanced getters by their index, i.e. `Get...ByID`
- start iteration callbacks with `Iterate...`

It's not perfect. Personally, I'd prefer calling getters by their entity, and setters by their function, i.e. `BridgeLimit`, `BridgeLimits`, `BridgeLimitByID`, `UpdateBridgeLimit`, etc...

But the thoughts outlined above seem to be more consistent with the cosmos SDK coding style. Just some food for thought here.